guides/doc-Upgrading_and_Updating/master.adoc

```diff
@@ -68,33 +68,37 @@ include::topics/upgrading_content_hosts.adoc[leveloffset=+2]
 endif::[]
 
 //Post-Upgrade Tasks
-include::topics/post_upgrade_tasks.adoc[leveloffset=+2]
+include::common/modules/con_performing-post-upgrade-tasks.adoc[leveloffset=+2]
 
 // Upgrading Discovery
-include::topics/upgrading_discovery_parent.adoc[leveloffset=+2]
+include::topics/upgrading_discovery_parent.adoc[leveloffset=+3]
 
 // Upgrading Discovery on Satellite Server
-include::topics/upgrading_discovery_satellite.adoc[leveloffset=+3]
+include::topics/upgrading_discovery_satellite.adoc[leveloffset=+4]
 
 // Configuring Subnets with a Template Capsule
-include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+3]
+include::topics/upgrading_discovery_subnet_with_a_template_capsule.adoc[leveloffset=+4]
 
 ifndef::foreman-deb[]
 // Upgrading virt-who
-include::topics/upgrading_virt_who.adoc[leveloffset=+2]
+include::topics/upgrading_virt_who.adoc[leveloffset=+3]
 endif::[]
 
 ifdef::satellite[]
 // Removing the Previous Version of the Satellite Tools Repository
-include::topics/removing_satellite_tools_repository.adoc[leveloffset=+2]
+include::topics/removing_satellite_tools_repository.adoc[leveloffset=+3]
+endif::[]
+
+ifdef::foreman-el,satellite[]
```

Comment:
Is this condition sufficiently broad?

Reply:
it at least also needs `katello`

Reply:
@maximiliankolb Do you want this included for orcharhino?

Reply:
Yes, please add orcharhino.